### ISSUE TITLE
Pre-training times: v2 vs. v3

### ISSUE THREAD

**Issue by stefan-it** [microsoft/DeBERTa#100]
Hi,

I would be very interesting to also see a comparison of pre-training times for DeBERTa v2 versus the recently released v3, that is using RTD.

The v2 paper mentioned pre-training times:

![image](https://user-images.githubusercontent.com/20651387/162736542-6600f23e-3ed7-4d00-9db0-6cf7998e571b.png)

But what about v3 base, large and multi-lingual models :thinking:

**Comment by WissamAntoun** [comment]
I was trying to pretrain DeBERTav2 with RTD objective (but without the Gradient-Disentangled Emb. sharing). I noticed that it runs way slower than electra (which is bert based).

I tried doing some quick benchmarking and noticed that deberta is twice as slow as bert for inference 
![image](https://user-images.githubusercontent.com/44616226/163189751-1bc3ca2d-7b95-4f8c-8879-07b0123aef4b.png)
